components/activity-section.jsx

```diff
@@ -0,0 +1,41 @@
+var React = require('react');
+var Illustration = require('../components/illustration.jsx');
+
+var ActivitySection = React.createClass({
+  propTypes: {
+    title: React.PropTypes.string.isRequired,
+    activities: React.PropTypes.array.isRequired
+  },
+  render: function () {
+    return (
+      <section className="activity-section">
+        <h2>
+          Section {this.props.title}
+        </h2>
+        {this.props.activities.map(function (activity, i) {
+          return (
+            <div className="activity" key={i}>
+              <Illustration height={122} width={122}
+              src1x={activity.image1x}
+              src2x={activity.image2x}
+              alt=""
+              link={activity.link}>
+                <div className="curriculum-description">
+                  <a href={activity.link}>
+                    <h3>{activity.title}</h3>
+                  </a>
+                  <p>
+                    <em>{activity.subtitle}</em>
+                  </p>
+                  <p className="description" dangerouslySetInnerHTML={{__html: activity.description}}></p>
```

Comment:
I think an alternative to :boom: **DANGEROUSLY** :boom: setting `innerHTML` here may be to actually set `activity.description` to a [`ReactElement`](https://facebook.github.io/react/docs/glossary.html#react-elements) rather than a string of HTML, e.g. 

```jsx
var activity = {
  title: "Cool Activity",
  description: <span>This activity is <em>cooooooooool</em>.</span>
};
```

Then you could actually just change line 30 to:

```jsx
<p className="description">{activity.description}</p>
```

In fact if `description` doesn't contain any markup, you could even leave it as a string, because JSX is cool that way. :sunglasses: 

I guess there probably aren't any performance benefits to doing this, but the JSX validation could help us ensure that there isn't any malformed HTML in the descriptions.

Um, that said, it could make it harder to move this kind of content into a DB and/or localize it in the future, so there's definitely trade-offs.

Meh, let's just leave it unchanged for now I guess. :expressionless: